core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -473,14 +481,25 @@ public TranslationStatus getTranslationObjectStatus(String strTranslationJobID,
           if (file.labels.contains("approval=REJECTED")) {
             rejected++;
           }
+          if (file.labels.contains("approval=COMPLETE")) {
+            rejected++;
+          }
         }
         boolean hasImported = imported > 0;
         boolean hasTranslated = translated > 0;
         boolean hasRejected = rejected > 0;
+        if (hasImported && hasTranslated && complete >= imported) {
+          log.warn("lilt: status for {} is COMPLETE", objectPath);
+          return TranslationStatus.COMPLETE;
+        }
         if (hasImported && hasTranslated && approved >= imported) {
           log.warn("lilt: status for {} is APPROVED", objectPath);
           return TranslationStatus.APPROVED;
         }
+        if (hasImported && hasTranslated && translated > rejected) {
+          log.warn("lilt: status for {} is APPROVED", objectPath);
```

Comment:
Shouldn't the log message say "... TRANSLATED"?

Also, what's the difference between this check and the one few lines below: `if (hasImported && hasTranslated)`?